[1.20.0] ZTS build is broken (PHP 5)

```
In file included from /usr/include/php-zts/php/Zend/zend_alloc.h:27:0,
                 from /usr/include/php-zts/php/Zend/zend.h:252,
                 from /usr/include/php-zts/php/main/php.h:35,
                 from /builddir/build/BUILD/php-pecl-grpc-1.20.0/ZTS/src/php/ext/grpc/php_grpc.h:29,
                 from /builddir/build/BUILD/php-pecl-grpc-1.20.0/ZTS/src/php/ext/grpc/php_grpc.c:19:
/builddir/build/BUILD/php-pecl-grpc-1.20.0/ZTS/src/php/ext/grpc/php_grpc.c: In function 'apply_ini_settings':
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:164:57: error: 'tsrm_ls' undeclared (first use in this function)
 #define TSRMG(id, type, element) (((type) (*((void ***) tsrm_ls))[TSRM_UNSHUFFLE_RSRC_ID(id)])->element)
                                                         ^
/builddir/build/BUILD/php-pecl-grpc-1.20.0/ZTS/src/php/ext/grpc/php_grpc.h:84:19: note: in expansion of macro 'TSRMG'
 #define GRPC_G(v) TSRMG(grpc_globals_id, zend_grpc_globals *, v)
                   ^
/builddir/build/BUILD/php-pecl-grpc-1.20.0/ZTS/src/php/ext/grpc/php_grpc.c:207:7: note: in expansion of macro 'GRPC_G'
   if (GRPC_G(enable_fork_support)) {
       ^
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:164:57: note: each undeclared identifier is reported only once for each function it appears in
 #define TSRMG(id, type, element) (((type) (*((void ***) tsrm_ls))[TSRM_UNSHUFFLE_RSRC_ID(id)])->element)
                                                         ^
/builddir/build/BUILD/php-pecl-grpc-1.20.0/ZTS/src/php/ext/grpc/php_grpc.h:84:19: note: in expansion of macro 'TSRMG'
 #define GRPC_G(v) TSRMG(grpc_globals_id, zend_grpc_globals *, v)
                   ^
/builddir/build/BUILD/php-pecl-grpc-1.20.0/ZTS/src/php/ext/grpc/php_grpc.c:207:7: note: in expansion of macro 'GRPC_G'
   if (GRPC_G(enable_fork_support)) {
       ^
make: *** [src/php/ext/grpc/php_grpc.lo] Error 1
make: *** Waiting for unfinished jobs....

```

--- a/src/php/ext/grpc/php_grpc.c
+++ b/src/php/ext/grpc/php_grpc.c
@@ -203,7 +203,7 @@ void register_fork_handlers() {
   }
 }
 
-void apply_ini_settings() {
+void apply_ini_settings(TSRMLS_D) {
   if (GRPC_G(enable_fork_support)) {
     char *enable_str = malloc(sizeof("GRPC_ENABLE_FORK_SUPPORT=1"));
     strcpy(enable_str, "GRPC_ENABLE_FORK_SUPPORT=1");
@@ -392,7 +392,7 @@ PHP_MINFO_FUNCTION(grpc) {
  */
 PHP_RINIT_FUNCTION(grpc) {
   if (!GRPC_G(initialized)) {
-    apply_ini_settings();
+    apply_ini_settings(TSRMLS_C);
     grpc_init();
     register_fork_handlers();
     grpc_php_init_completion_queue(TSRMLS_C);

